src/main/kotlin/no/nav/syfo/db/DatabaseUtil.kt

```diff
@@ -6,12 +6,16 @@ import no.nav.syfo.db.domain.UtsendtVarsel
 import no.nav.syfo.kafka.consumers.syketilfelle.domain.KSyketilfellebit
 import no.nav.syfo.syketilfelle.domain.Syketilfellebit
 import no.nav.syfo.syketilfelle.domain.tagsFromString
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
 import java.sql.Date
 import java.sql.ResultSet
 import java.sql.Timestamp
 import java.time.LocalDateTime
 import java.util.*
 
+val log: Logger = LoggerFactory.getLogger("no.nav.syfo.db")
```

Comment:
Er denne i bruk?

Reply:
@kaja-nav Ja, brukes i SyketilfelleDAO. Tenkte det er greit å definere den her, så slipper man gjøre det i andre DAO klasser hvis man får bruk for den andre steder.

Reply:
Jeg foretrekker å ha en pr klasse. Jeg tror ikke personlig jeg kommer til å huske på at det ligger en felles-log i DatabaseUtil, siden det ikke er slik vi vanligvis gjør det, i tillegg at det kan blir vanskeligere å feilsøke fordi man ikke nødvendigvis skjønner hvor log-statementene er gjort:
<img width="1032" alt="image" src="https://user-images.githubusercontent.com/75254007/186852146-e3ab1b25-fb2f-4c66-80f8-8a6c7ed1a203.png">

Reply:
Bildet er fra kibana, forresten. Var litt lite context med bildet 😄

Reply:
Fikset 👍